src/routes/meetups.ts

```diff
@@ -18,6 +18,27 @@ export const meetups: TMeetup[] = [
 	// 		},
 	// 	],
 	// },
+	{
+		dateISO: '2025-04-28T18:00:00.000Z',
```

Comment:
It's probably not 18:00 with the ISO stuff
Other than that, looks good. Thanks for doing it 👍

Reply:
Right we are +2 #fixed